File: num2.go
package main
import "fmt"
func main() {
	var n, m int
	fmt.Print("full bottles: ")
	fmt.Scan(&n)
	fmt.Print("empty bottles for exchange: ")
	fmt.Scan(&m)

	mc := n
	mcount := n
	count := 0
	ost := 0
	drinkedBottles := 0

	for mcount > 0 {
        for i := 0; i < n; i++{ 
            fmt.Printf("F")   
        }
        fmt.Print("\n")
		drinkedBottles += n
		count++
		mc = n + ost
	 for i := 0; i < mc; i++{ 
            fmt.Printf("E")   
        }
        fmt.Print("\n")

		ost = mc % m
		n = mc / m
		count++
		mc = ost
		mcount = n

File: num2.cs
using System;

class Program
{
    static void Main()
    {
        int n = 0;
        Console.Write("full bottles: ");
        n = int.Parse(Console.ReadLine());

        int m = 0;
        Console.Write("empty bottles for exchange: ");
        m = int.Parse(Console.ReadLine());

        int mc = n;
        int mcount = n;
        int count = 0;
        int ost = 0;
        int drinkedBottles = 0;

        while (mcount > 0)
        {
            Console.WriteLine(new string('F', n));
            drinkedBottles += n;
            count++;
            mc = n + ost;
            Console.WriteLine(new string('E', mc));
            ost = mc % m;
            n = mc / m;
            count++;
            mc = ost;
            mcount = n;
        }

        Console.WriteLine(drinkedBottles + " " + (count - 1));
    }
}
